api_v3/lib/types/filters/KalturaUserEntryFilter.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @package api
+ * @subpackage filters
+ */
+class KalturaUserEntryFilter extends KalturaUserEntryBaseFilter
+{
+	/**
+	 * @return baseObjectFilter
+	 */
+	protected function getCoreFilter()
+	{
+		// TODO: Implement getCoreFilter() method.
+		return new UserEntryFilter();
+	}
+
+	/**
+	 * @param KalturaFilterPager $pager
+	 * @param KalturaDetachedResponseProfile $responseProfile
+	 * @return KalturaListResponse
+	 */
+	public function getListResponse(KalturaFilterPager $pager, KalturaDetachedResponseProfile $responseProfile = null)
+	{
+
+		$response = new KalturaUserEntryListResponse();
+		if ( in_array(kCurrentContext::getCurrentSessionType(), array(kSessionBase::SESSION_TYPE_NONE,kSessionBase::SESSION_TYPE_WIDGET)) )
+		{
+			$response->totalCount = 0;
+			return $response;
+		}
+
+		$c = new Criteria();
+		$userEntryFilter = new UserEntryFilter();
+		$this->toObject($userEntryFilter);
+		$userEntryFilter->attachToCriteria($c);
+
+		if (!is_null($pager))
+		{
+			$pager->attachToCriteria($c);
+		}
+
+		$list = UserEntryPeer::doSelect($c);
+
+		$response->totalCount = UserEntryPeer::doCount($c);
```

Comment:
it would be best to use this pattern - https://github.com/kaltura/server/blob/Jupiter-10.14.0/api_v3/lib/types/conversionProfile/filters/KalturaAssetFilter.php#L81
in order to avoid the doCount when possible